C/plugins/south/dht11/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 2.4.0)
+
+project(dht11)
+
+set(CMAKE_CXX_FLAGS_DEBUG "-O0 -ggdb")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(WIRINGPILIBS -lwiringPi -lwiringPiDev)
+
+# Find source files
+file(GLOB SOURCES *.cpp)
+
+# Include header files
+include_directories(include ../../../services/common/include ../common/include ../../../common/include ../../../thirdparty/rapidjson/include /usr/local/include/)
+
+file(GLOB SOURCES "*.cpp" ../common/*.cpp)
```

Comment:
could you please explain with comments w.r.t. 
```
+# Find source files
+file(GLOB SOURCES *.cpp)
```

Reply:
set SOURCES variable to list of files in current directory. This is being done twice in CMakeLists.txt, so  removing one of them.